docs/lib/getting-started/fragments/android/setup.md

```diff
@@ -1,78 +1,108 @@
+    ```
 
-* [Install Android SDK with a minimum API level of 16 (Jelly Bean).](https://developer.android.com/studio/releases/platforms)
+## Set up your application
 
-* This guide assumes that you are familiar with Android development and tools. If you are new to Android development, you can follow [these steps](https://developer.android.com/training/basics/firstapp/creating-project) to create your first Android application using Java.
+### Create a new Android application
 
-* Install the latest version of the Amplify CLI by running:
+1. Open **Android Studio**. Select **+ Start a new Android Studio project**.
 
-```bash
-npm install -g @aws-amplify/cli
-```
+    ![](~/images/lib/getting-started/android/set-up-android-studio-welcome.png)
 
-## Step 1: Configure your app
-You can use an existing Android app or create a new one by following [these steps](https://developer.android.com/training/basics/firstapp/creating-project).
+1. In **Select a Project Template**, select **Empty Activity**. Press **Next**.
 
-a. Open your **project** `build.gradle` and add the following:
-* `mavenCentral()` as a repository
-* `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'` as a dependency
-* A plugin `'com.amplifyframework.amplifytools'` as shown in the example below:
+    ![](~/images/lib/getting-started/android/set-up-android-studio-select-project-template.png)
 
-```gradle
-buildscript {
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.1'
-        classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'
+1. Next, configure your project:
+
+    - Enter *Todo* in the **Name** field
+    - Select either *Java* or *Kotlin* from the **Language** dropdown menu
+    - Select *API 16: Android 4.1 (Jelly Bean)* from the **Minimum SDK** dropdown menu
+    - Press **Finish**
+
+  ![](~/images/lib/getting-started/android/set-up-android-studio-configure-your-project.png)
+
+Android Studio will open your project with a tab opened to either *MainActivity.java* or *MainActivity.kt* depending upon if you created a Java or Kotlin project respectively.
+
+![](~/images/lib/getting-started/android/set-up-android-studio-successful-setup.png)
+
+### Add Amplify to your application
+
+Amplify for Android is distribued as an Apache Maven package. In this section, you'll add the packages and other required directives to your build configuration.
+
+1. Expand **Gradle Scripts** and open **build.gradle (Project: Todo)**. Add the following lines:
+
+    - Add the line `classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'` within the `dependencies` block
+    - Add the line  `plugin 'com.amplifyframework.amplifytools'` at the end of the file 
+
+    ```groovy
+    buildscript {
+        ...
+
+        dependencies {
+            classpath 'com.android.tools.build:gradle:3.6.3'
+            classpath 'com.amplifyframework:amplify-tools-gradle-plugin:0.2.1'
+        }
```

Comment:
Is amplify tools really needed here?  I've had a really poor developer experience using amplify tools.  As far as I know, it is a script that gets invoked during the build process that has the ability to automatically run amplify push or generate models.  From my stand point, it doesn't provide too much value, hides what is happening, cumbersome to configure (which makes me feel like I don't have control over what's happening), and difficult to debug when things go wrong (which is often).